site/content/contribute/desktop/developer-setup.md

```diff
@@ -12,12 +12,46 @@ subsection: "Desktop App"
 - NPM
 - Git
 
+### Windows
+Recommended way to install prerequisites on windows is to install chocolatey package manager.
+once installed run from an administrator powershell console
+```
+choco install nvm
+```
+```
+choco install git
+```
+from a non privileged powershell console:
+```
+nvm install 10.18.1
+```
+
 ## Installing dependencies
 After installation, dependencies of `src/` directory are also installed.
 
+### OSX & Linux
 ```
 $ npm install
 ```
+### Windows
+on an administrator powershell run
+
+```
+> .\scripts\Makefile.ps1 Install-Deps
+```
+
+#### Known problems
+
+##### Policy
+If the above command fails, it might be due to not having the right policy to install. In that case use this command to temporarily change the policy:
```

Comment:
```suggestion
If the above command fails, it might be due to not having the right policy to install. You can temporarily change the policy using:
```